packages/flame/lib/src/components/isometric_tile_map_component.dart

```diff
@@ -105,32 +128,40 @@ class IsometricTileMapComponent extends PositionComponent {
 
   /// Same as getBlockRenderPosition but the arguments are exploded as integers.
   Vector2 getBlockRenderPositionInts(int i, int j) {
-    final halfTile = effectiveTileSize / 2;
+    final halfTile = Vector2(effectiveTileSize.x / 2, (effectiveTileSize.y / 2) / scalingFactor);
     final pos = Vector2(i.toDouble(), j.toDouble())..multiply(halfTile);
     return cartToIso(pos) - halfTile;
   }
 
+  /// Same as [getBlockRenderPositionInts] but is used for props ie. elements with <= -2 values
+  ///
+  /// This accounts for difference in size of the propSize.
+  Vector2 getVariableRenderPositionInts(int i, int j) {
+    final initialPosition = getBlockRenderPositionInts(i, j);
+    initialPosition.sub(Vector2(pSize.x - effectiveTileSize.x, pSize.y - effectiveTileSize.y));
```

Comment:
Use `..x -=` and `..y -=`, or `.setValues(initialPosition.x - ..., initialPosition.y - ...)`